config.go

```diff
@@ -44,7 +44,7 @@ type config struct {
 	Listen          string        `long:"listen" ini-name:"listen" description:"The ip:port to listen for API requests."`
 	LogLevel        string        `long:"loglevel" ini-name:"loglevel" description:"Logging level." choice:"trace" choice:"debug" choice:"info" choice:"warn" choice:"error" choice:"critical"`
 	Network         string        `long:"network" ini-name:"network" description:"Decred network to use." choice:"testnet" choice:"mainnet" choice:"simnet"`
-	VSPFee          float64       `long:"vspfee" ini-name:"vspfee" description:"Fee percentage charged for VSP use. eg. 2.0 (2%), 0.5 (0.5%)."`
+	VSPFee          float64       `long:"vspfee" ini-name:"vspfee" description:"Flat fee charged for VSP use. i.e 7.5."`
```

Comment:
`i.e. 7.5`? Is the flat fee supposed to always be 7.5? If so, why is this a config (read _configurable_) option?